core/src/main/java/nl/nn/adapterframework/util/AppConstants.java

```diff
@@ -192,19 +194,41 @@ public String getResolvedProperty(String key) {
 	}
 
 	/**
-	 * Creates a tokenizer from the resolved value of this key. As a separator the "," is used.
-	 * Uses the {@link #getResolvedProperty(String)} method.
-	 * Can be used to process lists of values.
+	 * Retrieves a list property value associated with the specified key. The method first resolves the property value using the {@link #getResolvedProperty(String)} method. If the resolved property value is null, an empty list is returned.
+	 *
+	 * @param key the key of the property value to retrieve
+	 * @return a list of string values associated with the specified key, or an empty list if the resolved property is null
 	 */
-	public StringTokenizer getTokenizedProperty(String key) {
-		return new StringTokenizer(getResolvedProperty(key), ",");
+	public List<String> getListProperty(String key) {
+		String resolvedProperty = getResolvedProperty(key);
+		if (resolvedProperty == null) {
+			return Collections.emptyList();
+		}
+		return StringUtil.split(resolvedProperty);
 	}
+
+	/**
+	 * Retrieves a list property value associated with the specified key. The method first resolves the property value using the {@link #getResolvedProperty(String)} method. If the resolved property value is null, it returns the list of string values provided as "defaults".
+	 *
+	 * @param key the key of the property value to retrieve
+	 * @param defaults the default list of string values to return if the resolved property is null
+	 * @return a list of string values associated with the specified key, or the default list if the resolved property is null
+	 */
+	public List<String> getListProperty(String key, @Nonnull String defaults) {
+		String list = getResolvedProperty(key);
+		if (list != null) {
+			return StringUtil.split(list);
+		}
+		return StringUtil.split(defaults);
```

Comment:
is het bij deze 2 methodes niet mooier ze te overloaden?

Reply:
Dat waren ze al niet maar kan op zich wel.